src/tables/tableRows.js

```diff
@@ -50,7 +50,6 @@ export function generateLocalApplicationRows(
         {
           "data-test-column": "version",
           content: version,
-          className: "u-align--right",
```

Comment:
Removed the right alignment as versions are not numerical

Reply:
This value is user defined so it 'can' be a number. But I think there is a bug here...you'll see the app has a workload version defined but it's not rendered in the UI.

![Screen Shot 2021-04-13 at 3 31 44 PM](https://user-images.githubusercontent.com/532033/114624157-7e580c80-9c6d-11eb-9708-000d1a85b552.png)

Reply:
Should we also truncate these versions and have tooltips with the full value if they can be that long in K8s land?

Reply:
Mentioned under the screenshot that once the other branch lands it will truncate the version. The title would be a good follow up branch.

Reply:
Well it could be in odd occasions but you wouldn't define is a number type. Even 2.3.4 is not a number.